mm2src/coins/eth/web3_transport/websocket_transport.rs

```diff
@@ -357,7 +356,7 @@ async fn send_request(
         serialized_request = serde_json::to_string(&wrapper)?;
     }
 
-    let mut tx = transport.controller_channel.tx.lock().await;
+    let mut tx = transport.controller_channel.tx.clone();
```

Comment:
we might want to move this var down just before it's used

Reply:
[6eb6fc3](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2291/commits/6eb6fc3ad97b812d4029adf20742655d7022e408)